web/js/components/timeline/timeline-controls/date-change-arrows.js

```diff
@@ -89,11 +91,16 @@ class DateChangeArrows extends PureComponent {
         <div
           className={`button-action-group ${leftArrowDisabled ? 'button-disabled' : ''}`}
           id="left-arrow-group"
-          title="Click and hold to animate backwards"
           onMouseDown={this.leftArrowDown}
           onMouseUp={this.leftArrowUp}
           onMouseLeave={this.leftArrowUp}
         >
+          <HoverTooltip
+            isMobile={isMobile}
+            labelText="Click and hold to animate backwards"
```

Comment:
I wonder if this tooltip should include the primary interaction as well?  Though that would lead to a rather large tooltip.  Only telling the user how to click and hold to "animate" may mislead them to think that is the primary action of these buttons.

Reply:
As is, it's a little too much text for me too. I reused the "Click and hold to animate backwards" text but it almost sounds more like a super user/extra app features kind of tip vs general functionality I'd expect on hover.